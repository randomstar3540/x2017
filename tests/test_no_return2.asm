FUNC LABEL 0
    CAL VAL 2
    RET
FUNC LABEL 5
    RET
FUNC LABEL 2
    CAL VAL 3
FUNC LABEL 3
    CAL VAL 4
FUNC LABEL 4
    CAL VAL 5
