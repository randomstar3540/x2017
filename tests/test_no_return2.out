ERROR: Function terminated without return
